gremlin-archetype/gremlin-archetype-server/src/main/resources/archetype-resources/src/main/resources/gremlin-server.yaml

```diff
@@ -0,0 +1,45 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
since `gremlin-server` is a `test` scope dependency, this should move over to `.../src/test/resources` in the generated project